pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: platform
-version: 2.2.0
+version: 2.2.1
```

Comment:
FYI for future changes. This should have been `2.3.0`. Changes which introduce new API surface area should always get at least a feature version bump.